apps/wallet-ui/next.config.mjs

```diff
@@ -1,4 +1,14 @@
 /** @type {import('next').NextConfig} */
-const nextConfig = {};
+const nextConfig = {
+  images: {
+    dangerouslyAllowSVG: true,
+    remotePatterns: [
+      {
+        protocol: "https",
+        hostname: "**",
```

Comment:
this feels potentially dangerous, it means anyone can use this project as a free (not for us!) image resizing service potentially, no?

Reply:
Is there a better way to display the image for any NFT?